extensions/window/layout.lua

```diff
@@ -902,7 +902,7 @@ function layout.applyLayout(rules) layout.new(rules):apply():delete() end
 
 
 --TODO...
-function layout.getLayout(layout,includeScreen)
```

Comment:
Since this whole function is `TODO`, I'm not sure it's a good idea to remove its arguments at this point.